tests/testthat/indention_curly_brackets/multi_line_curly_only-out.R

```diff
@@ -0,0 +1,4 @@
+{
+  {1 + 3}
```

Comment:
The spaces are added due to the first `{`. `transform_indent_curly()` adds braces  to everything between two curly braces.